### ISSUE TITLE
Just completely stopped working today? preferences field for "Shortcut character" is blank and refuses input... no GUI icon either?

### ISSUE THREAD

**Issue by OwenAR** [dgutov/bmreplace#45]
It just completely stopped working today.
I note that the field for "Shortcut character" under preferences is blank,
and it refuses to take input.
Like, trying to click on the field with the mouse just puts a sort of dotted line box inside of the field,
I guess showing focus but that's all.

I always used the hotkey shortcut,
but I think there should be a GUI icon in the menu/toolbars as well, right?
But I can't find one at all.

I have no idea how to diagnose this to get any more information beyond that, sorry...

**Comment by OwenAR** [comment]
Oh, right, durr, it's under `bmreplace` in `about:config`...

```
extensions.bmreplace.keep-title-default;true
extensions.bmreplace.shortcut-key;b
extensions.bmreplace.shortcut-modifiers;accel,alt
extensions.bmreplace.version;1.6.3
extensions.bmreplace@dgutov.github.com.install-event-fired;true
```

so... from that it looks like it should work as normal?

Of course I tried setting the hotkey to other things (alt+shift+b etc), but no...

(Also, `browser.uiCustomization.state` *does* mention `bmreplace-button` in the list of things that should be placed...)

**Comment by OwenAR** [comment]
Re-installing it again manages to get the GUI icon back,
but the problem with the hotkey is still the same...
As far as I can think, haven't done *any* updates or installed any new things in the past few days that could be interacting with it...

Okay, wait, re-installing it again *did* add a bunch of entries to about:config...
but nothing relevant?

And then I tried manually re-entering:
```
extensions.bmreplace.shortcut-key;b
extensions.bmreplace.shortcut-modifiers;accel,alt
```
like, I just selected "Modify" and then re-entered *exactly the same string that was already in the field*...

And now it works again!

The input field in the preferences found under about:addons was still broken,
but reopening that fixed it
(I see now that it's actually a dropdown menu that wasn't working before, not a text field).

This was... a really weird, random, transient out-of-nowhere bug?

So... I guess just ignore this, and unless someone else ever reports something like it,
we'll just have to assume it was random transient filesystem corruption on my computer or something?

Thank you rubber ducky. :/
